terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -200,22 +200,37 @@
     'hannes@ucsc.edu'
 )
 
+operator_keys = {
+    'achave11@ucsc.edu': (
+        'ssh-rsa'
+        ' '
+        'AAAAB3NzaC1yc2EAAAADAQABAAABgQCrIU25zlzHBxIdEATJZsGXvatdWuen5zlOw1uE25spQ8eNnOUfbz5fR'
+        'yiQqyMNxE/dX2hCCDT1mr5Flke4uJ0FayC/l5ZC3bKYE2gnILbZBNsFuueZuDy9pRmZ+eTYs3vKXN361+loRi'
+        '6ag8h/pOQCvx6oO5NrVSBse0NcEn1tk1h7C1hOf8sblW17+OO9aDQJAA7G4PJw2kBRCYYEwDNLBRy3k1wBdcK'
+        'G2t2SuVh+PCpmMPA5/i/raDUqATO1H3bcRubtyGHNbAtihL5HLZK83O9fHVf/MD7il4N/9OwBNpOwvc2gi9zp'
+        'ChGpbl5jA2ZfoEDEOhX4ffOD1UwmkmkoUC82BvHyAwdnqgh3Nk4qCum53TsMhXVWMW/8tr/t+AxjE3/Acwj6H'
+        'VMz2j+67A0p1oaTbxBXdf00BmAYV2xPZNg8Fa2/AkQWPt4c4JJnktVjWM8/PU1h6FamyHfQ6pNmi+j6rHz9UZ'
+        'e1Zt6WybGr+Tt+KifhbCnZQkg74I1uT6M='
+        ' '
+        'achave11@ucsc.edu'
+    ),
+    'dsotirho@ucsc.edu': (
+        'ssh-rsa'
+        ' '
+        'AAAAB3NzaC1yc2EAAAADAQABAAABAQDDPUVio1tlAstsaM2Da7QfSIv0zMU7JwjO7a/BvsWg0tXES'
+        'gpL59i5QcycpYq6q7naF+N0co325e/OJ4lzi13T5xojSbh/kNETwiI+aJ9f0GxwnygcvVUpsTlH3X01fR+1xm'
+        'rlGWi8AhEfbFyAFaqb2i+Whbkt9/oa3EIv4l+OSH6VSRtKRE56IvJ06hnWQ3yR57wxRBnHjiUuEBQ5I0jsye3'
+        '0OE0USvjfbHqjbR9zyKCgnGuf/fY4aC+oimHu6/FSS3Q8+f5BtRrUjcYvddbAHnzrx08csztCx3s7iA5qUdhr'
+        'W07wIjyG7vfB9Y70CDNsfi1Zo/Ff+IMKSzPtasXx'
+        ' '
+        'dsotirho@ucsc.edu'
+    )
+}
+
 other_public_keys = {
-    'dev': [
-        (
-            'ssh-rsa'
-            ' '
-            'AAAAB3NzaC1yc2EAAAADAQABAAABgQCrIU25zlzHBxIdEATJZsGXvatdWuen5zlOw1uE25spQ8eNnOUfbz5fR'
-            'yiQqyMNxE/dX2hCCDT1mr5Flke4uJ0FayC/l5ZC3bKYE2gnILbZBNsFuueZuDy9pRmZ+eTYs3vKXN361+loRi'
-            '6ag8h/pOQCvx6oO5NrVSBse0NcEn1tk1h7C1hOf8sblW17+OO9aDQJAA7G4PJw2kBRCYYEwDNLBRy3k1wBdcK'
-            'G2t2SuVh+PCpmMPA5/i/raDUqATO1H3bcRubtyGHNbAtihL5HLZK83O9fHVf/MD7il4N/9OwBNpOwvc2gi9zp'
-            'ChGpbl5jA2ZfoEDEOhX4ffOD1UwmkmkoUC82BvHyAwdnqgh3Nk4qCum53TsMhXVWMW/8tr/t+AxjE3/Acwj6H'
-            'VMz2j+67A0p1oaTbxBXdf00BmAYV2xPZNg8Fa2/AkQWPt4c4JJnktVjWM8/PU1h6FamyHfQ6pNmi+j6rHz9UZ'
-            'e1Zt6WybGr+Tt+KifhbCnZQkg74I1uT6M='
-            ' '
-            'achave11@ucsc.edu'
-        ),
-    ],
+    'dev': [operator_keys.values()],
```

Comment:
This causes a TypeError. So it looks like this round wasn't tested. Also, the use of a dictionary does not make much sense in this version. Why create a dictionary if only the values are used? I had something else in mind but I realize now that not using a dictionary at all is simpler:

```
Index: terraform/gitlab/gitlab.tf.json.template.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/terraform/gitlab/gitlab.tf.json.template.py b/terraform/gitlab/gitlab.tf.json.template.py
--- a/terraform/gitlab/gitlab.tf.json.template.py	(revision 05d25d92e0a801f4c4490140a754f0926eb602bc)
+++ b/terraform/gitlab/gitlab.tf.json.template.py	(date 1685164799962)
@@ -200,8 +200,8 @@
     'hannes@ucsc.edu'
 )
 
-operator_keys = {
-    'achave11@ucsc.edu': (
+operator_keys = [
+    (
         'ssh-rsa'
         ' '
         'AAAAB3NzaC1yc2EAAAADAQABAAABgQCrIU25zlzHBxIdEATJZsGXvatdWuen5zlOw1uE25spQ8eNnOUfbz5fR'
@@ -214,7 +214,7 @@
         ' '
         'achave11@ucsc.edu'
     ),
-    'dsotirho@ucsc.edu': (
+    (
         'ssh-rsa'
         ' '
         'AAAAB3NzaC1yc2EAAAADAQABAAABAQDDPUVio1tlAstsaM2Da7QfSIv0zMU7JwjO7a/BvsWg0tXES'
@@ -225,12 +225,12 @@
         ' '
         'dsotirho@ucsc.edu'
     )
-}
+]
 
 other_public_keys = {
-    'dev': [operator_keys.values()],
-    'anvildev': [operator_keys.values()],
-    'anvilprod': [operator_keys.values()],
+    'dev': operator_keys,
+    'anvildev': operator_keys,
+    'anvilprod': operator_keys,
     'prod': []
 }
```